mobileconnectsdk/Mobile Connect SDK 16.4 Release Notes.md

```diff
@@ -0,0 +1,25 @@
+# Mobile Connect SDK 16.4 Release Notes
+
+Version 16.4 of the Mobile Connect SDK adds support for two main features:
+
+- Receive real-time Salto events for offline Salto readers on remote locations and transmit them to Command Centre and Salto servers via the Gallagher cloud without the need for badging at Salto update points.
```

Comment:
I guess we took this from somewhere but why 'real-time'? The events are historical at the point they're being uploaded to the cloud.